stubs/html5lib/html5lib/html5parser.pyi

```diff
@@ -32,6 +32,7 @@ class HTMLParser:
     lastPhase: Any
     beforeRCDataPhase: Any
     framesetOK: bool
+    tokenizer: Any
```

Comment:
This is set by [HTMLParser._parse](https://github.com/html5lib/html5lib-python/blob/master/html5lib/html5parser.py#L127), not by the constructor, so it's not always guaranteed to be there, but the same is true of `lastPhase`, `beforeRCDataPhase`, `framesetOK`, and so on.